reporters_db/data/reporters.json

```diff
@@ -16922,7 +16993,8 @@
                 "Pa.": {
                     "end": null,
                     "regexes": [
-                        "(?P<volume>\\d+|81 ?\\*|81 1/2|\\* ?81|81 ?½) $reporter,? $page"
+                        "(?P<volume>\\d+|81 ?\\*|81 1/2|\\* ?81|81 ?½) $reporter,? $page",
```

Comment:
do we have an example with `½`

Reply:
I couldn't find any with "½" I only found with 1/2: 81 1/2 Pa. 9

Reply:
Okay. so let's drop the 1/2 key stroke unless its actually used.

Reply:
Can we push this thru?